scripts/release/install_release_script_dependencies.R

```diff
@@ -55,6 +55,7 @@ warnings()
 
 library("devtools")
 install_github("datashield/dsBaseClient")
-install_github("datashield/dsMediationClient")
+install_github("https://github.com/transbioZI/dsMTLClient")
```

Comment:
Are there no CRAN versions of these? Guess not ... is it possible to add version argument for `install_github`

Reply:
No CRAN versions but I agree - I've added versions for both packages

Reply:
:)